.github/build-image.yaml

```diff
@@ -0,0 +1,38 @@
+name: Release cloud-dev-hack-blog-internal-api
+on:
+  push:
+    branches:
+      - main
```

Comment:
looks like this repo has a `master` branch, not `main`, this of course has to match that name. You can rename that branch to `main` (that's the new standard).